README.md

```diff
@@ -555,20 +554,12 @@ $pagerfantaHtml = $myView2->render($pagerfanta, $routeGenerator);
 $pagerfantaHtml = $myView2->render($pagerfanta, $routeGenerator, array('next_message' => 'Siguiente!!'));
 ```
 
-## Todo
-
-## Author
-
-Pablo Díez - <pablodip@gmail.com>
-
-## License
-
-Pagerfanta is licensed under the MIT License. See the LICENSE file for full details.
+## Acknowledgements
 
-## Sponsors
+Pagerfanta is inspired by [Zend Paginator](https://github.com/zendframework/zf2).
 
-[WhiteOctober](http://www.whiteoctober.co.uk/)
+Thanks also to Pablo Díez (pablodip@gmail.com) for most of the work on the first versions of Pagerfanta.
 
-## Acknowledgements
+## Licence
 
-Pagerfanta is inspired by [Zend Paginator](https://github.com/zendframework/zf2).
+Pagerfanta is licensed under the [MIT License](LICENCE).
```

Comment:
License with an "s" is a verb; with a "c" it's a noun (although in American English they only use "license").